prow/prowjobs/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp/secrets-store-csi-driver-provider-gcp-config.yaml

```diff
@@ -9,13 +9,3 @@ presubmits:
       - image: gcr.io/google.com/cloudsdktool/cloud-sdk:latest
```

Comment:
So long as you make the default k8s SA in the above cluster context use workload identity this works great.

Otherwise you'll need a `serviceAccountName` on this podspec